logging/cloudwatch/hook.go

```diff
@@ -173,11 +172,21 @@ func (h *Hook) sendBatch(batch []*cloudwatchlogs.InputLogEvent) {
 		SequenceToken: h.nextSequenceToken,
 	}
 	resp, err := h.svc.PutLogEvents(params)
-	if err != nil {
-		h.err = &err
-	} else {
-		h.nextSequenceToken = resp.NextSequenceToken
+	if err == nil {
+		h.NextSequenceToken = resp.NextSequenceToken
```

Comment:
I think this needs to be:
`
h.nextSequenceToken = resp.NextSequenceToken
`
I don't think h has a NextSequenceToken member variable.  Maybe I'm overlooking it?